fmriprep/utils/transforms.py

```diff
@@ -36,57 +37,76 @@ def load_transforms(xfm_paths: list[Path], inverse: list[bool]) -> nt.base.Trans
     return chain
 
 
-def load_ants_h5(filename: Path) -> nt.TransformChain:
-    """Load ANTs H5 files as a nitransforms TransformChain"""
-    affine, warp, warp_affine = parse_combined_hdf5(filename)
-    warp_transform = nt.DenseFieldTransform(nb.Nifti1Image(warp, warp_affine))
-    return nt.TransformChain([warp_transform, nt.Affine(affine)])
+FIXED_PARAMS = np.array([
+    193.0, 229.0, 193.0,  # Size
+    96.0, 132.0, -78.0,   # Origin
+    1.0, 1.0, 1.0,        # Spacing
+    -1.0, 0.0, 0.0,       # Directions
+    0.0, -1.0, 0.0,
+    0.0, 0.0, 1.0,
+])  # fmt:skip
 
 
-def parse_combined_hdf5(h5_fn, to_ras=True):
+def load_ants_h5(filename: Path) -> nt.TransformChain:
+    """Load ANTs H5 files as a nitransforms TransformChain"""
     # Borrowed from https://github.com/feilong/process
     # process.resample.parse_combined_hdf5()
-    h = h5py.File(h5_fn)
+    #
+    # Changes:
+    #   * Tolerate a missing displacement field
+    #   * Return the original affine without a round-trip
+    #   * Always return a nitransforms TransformChain
+    #
+    # This should be upstreamed into nitransforms
+    h = h5py.File(filename)
     xform = ITKCompositeH5.from_h5obj(h)
-    affine = xform[0].to_ras()
+
+    # nt.Affine
+    transforms = [nt.Affine(xform[0].to_ras())]
+
+    if '2' not in h['TransformGroup']:
+        return transforms[0]
```

Comment:
this is returning an np.array, not a TransformChain

Reply:
Actually, an `nt.Affine()`, but fair point. I'll relax the return type.